### PR TITLE
Add ropsten-compatible deployment configuration

### PR DESCRIPTION
**Pull request by vgeddes**
I set `DescendantsUntilFinalized` to 3 until we have a solution for https://github.com/Snowfork/polkadot-ethereum/issues/286.

### CODE REVIEW

File: parachain/runtime/src/lib.rs [hunk 1]
```diff
@@ -368,7 +368,7 @@ impl bridge::Config for Runtime {
 
 #[cfg(not(feature = "test-e2e"))]
 parameter_types! {
-	pub const DescendantsUntilFinalized: u8 = 35;
+	pub const DescendantsUntilFinalized: u8 = 3;
```
Comment (Rizziepit) [inline parachain/runtime/src/lib.rs:371]: This is quite dangerous for mainnet I think. But as a temporary thing, that's fine

Comment (vgeddes) [reply parachain/runtime/src/lib.rs:371]: yeah is a total temporary thing, as ropsten is our de-facto "live" environment until we actually deploy a production stack connected to mainnet

Comment (philipstanislaus) [reply parachain/runtime/src/lib.rs:371]: Can we add a ticket for that though?

Comment (philipstanislaus) [reply parachain/runtime/src/lib.rs:371]: Done actually: https://github.com/Snowfork/polkadot-ethereum/issues/291

**Pull request merged**